conf/config.local.user.php

```diff
@@ -1,13 +0,0 @@
-<?php
```

Comment:
This file was meant to store the user configuration, so it won't be overwritten each time.

What will be the right way to set them now?

Reply:
From what I read from the doc, `config.local.php` is the user configuration (this is where user can alter default settings). I don't know why we need `config.local.user.php` (mainely empty as it is).
I think we better store the data in the `data_dir` folder (`/home/yunohost.app/paheko`) so the data gets preserved at upgrade and is not deleted with the app (unless the `--purge` option is used)
`config.local.php` is preserved with `--keep` option or regenerated if needed.